### PR TITLE
Fixed logging to file for NLU server (issue #1554)

### LINKED ISSUE #1554: NLU server logging to files does not work
**Rasa NLU version**: all

**Operating system** (windows, osx, ...): OSX

**Issue**:

Logging to files with `--write` option doesn't work. Replacing the `_configure_logging()` method in `rasa_nlu.server` fixes the problem:
```python
@staticmethod
    def _configure_logging(loglevel, logfile):
        for handler in logging.root.handlers[:]:
            logging.root.removeHandler(handler)
        logging.basicConfig(filename=logfile,
                            level=loglevel)
        logging.captureWarnings(True)
```
I'm not sure if there's a better approach.

Also, there is no command line option for `--loglevel` which that function is supposed to use.

### PR DESCRIPTION
**Pull request by eric-bonfadini**
This fixes #1554 

**Proposed changes**:
- Added loglevel input parameter in CLI
- Fixed logging to file

**Status (please check what you already did)**:
- [x] made PR ready for code review
- [ ] added some tests for the functionality
- [ ] updated the documentation
- [ ] updated the changelog

### CODE REVIEW

**Comment by akelad** [comment]
thanks for submitting this PR, we'll give it a review as soon as possible

**Comment by wochinge** [comment]
@eric-bonfadini Thanks for your pr. A couple of questions:
* the commandline arguments `debug` and `verbose` are already writing to `loglevel`. Do you need a more detailed option or what's the purpose of the log level
* the removal of the handler is a bit brute force. We could also just check whether they used configured a log file using `write` and then decide whether we configure the colored logging or the file logging (utils.configure_colored_logging(args.loglevel)) 

What do you think?

**Comment by eric-bonfadini** [comment]
Hi @wochinge , thanks for your feedback!

- About the loglevel, you're right, I didn't notice the `utils.add_logging_option_arguments(parser)` line and I got confused by the original request in the issue. I guess debug and verbose are fine for most of the use cases (unless we need just errors or every log from trace onwards). We can maybe double-check with @ricwo if he meant something different, if not I can revert my modification.

- About the handler issue, I got the same results by moving `utils.configure_colored_logging(args.loglevel)` from `main` to `_configure_logging`, and calling it only if no logfile was passed. Let me know if that's fine to you.

File: rasa/nlu/cli/server.py [hunk 1]
```diff
@@ -22,6 +22,10 @@ def add_server_arguments(parser):
                              "provide this token as a query parameter")
     parser.add_argument('-w', '--write',
                         help='file where logs will be saved')
+    parser.add_argument('--loglevel',
```
Comment (wochinge) [inline rasa/nlu/cli/server.py]: @eric-bonfadini We can remove this one then, right?

Comment (eric-bonfadini) [reply rasa/nlu/cli/server.py]: Yes, right.

**Pull request merged**